lib/protobuf/field/bool_field.rb

```diff
@@ -21,10 +21,13 @@ def acceptable?(val)
       end
 
       def coerce!(val)
-        if val == 'true'
-          true
-        elsif val == 'false'
-          false
+        case val
+        when String then
+          if val == 'true'
+            true
+          elsif val == 'false'
```

Comment:
Perhaps we can move these to constants so we aren't creating new string objects purely for checking for true/false here.